test/dotnet/Integration/RedisInputBindingTests.cs

```diff
@@ -34,6 +35,7 @@ public async void Get_SuccessfullyGets()
                     await Task.Delay(TimeSpan.FromSeconds(1));
 
                     await multiplexer.CloseAsync();
+                    IntegrationTestHelpers.StopRedis(redisProcess);
```

Comment:
What will be the consequence if we failed to stop redis because an exception was thrown in previous statements? If it is troublesome to diagnose failures, consider dispose pattern?

Reply:
If stopping a Redis instance fails, then the next integration tests could fail because they might not be able to start their own redis instances,